sources/Renderer/QtRenderer/QtSFML.hpp

```diff
@@ -37,27 +37,30 @@ namespace ComSquare::Renderer
 		void enableCgramViewer();
 		//! @brief Action called when clicking on the enable DMA viewer button.
 		void enableRegisterViewer();
+		//! @brief Action called when clicking on the enable Tile viewer button
+		void enableTileViewer();
 
 		//! @brief Action called when clicking on the reset button.
 		void reset();
 
 		QtFullSFML(SNES &snes, QWidget* parent, const QPoint& position, const QSize& size, int frameRate = 0);
 		QtFullSFML(const QtFullSFML &) = delete;
 		QtFullSFML &operator=(const QtFullSFML &) = delete;
-		~QtFullSFML() override = default;
+		~QtFullSFML() = default;
 	};
 
-	//! @brief A SFML renderer inside a QT window.
+	//! @brief A SFML renderer inside a QT widget.
 	class QtSFML : public IRenderer {
 	private:
 		//! @brief The main window that the app reside on.
-		QMainWindow _window;
+		QWidget *_window;
```

Comment:
Maybe put this raw pointer to a smart one?

Reply:
This is a QWidget, Qt handles everything internally with news and deletes. We can't really do something for that.

Reply:
idk honestly so i won't change it